client/js/bar.js

```diff
@@ -59,4 +54,9 @@ function createBarChart(chartValues, chartCategories) {
   }
 }
 
-createBarChart(CHART_VALUES, CHART_CATEGORIES);
+GENRE_ANALYSIS.then((DATA) => {
```

Comment:
Don't think `DATA` is the best variable name here. Since you're getting a object, something like `genreAnalysisInfo` or something like that

Reply:
+1; this is a parameter of a function and not a constant, so it should be in `camelCase`